tests/unit/generate_detect_target_contour.py

```diff
@@ -0,0 +1,165 @@
+"""
+Helper functions for test_detect_target_contour.
+
+"""
+
+import cv2
+import math
+import numpy as np
+
+LANDING_PAD_COLOR = (100, 50, 50)  # blue color
+
+# Test functions use test fixture signature names and access class privates
+# No enable
+# pylint: disable=protected-access,redefined-outer-name
+
+
+class BoundingBox:
```

Comment:
Order the classes in the flow order:
* `LandingPadData`
* `NumpyImage`
* `BoundingBox`
* `LandingPadTestData`